integration-tests/equations/test_coupled_transport.py

```diff
@@ -0,0 +1,63 @@
+"""
+Tests the CoupledTransportEquation() class.
+Two tracers -- a mixing ratio and a density -- are transported using
+combinations of advective and conservative forms.
+
+
+"""
+
+from firedrake import norm
+from gusto import *
+import pytest
+
+
+def run(timestepper, tmax, f_end):
+    timestepper.run(0, tmax)
+    norm1 = norm(timestepper.fields("f1") - f_end) / norm(f_end)
+    norm2 = norm(timestepper.fields("f2") - f_end) / norm(f_end)
+    return (norm1 + norm2)/2
+
+
+@pytest.mark.parametrize("geometry", ["slice", "sphere"])
+@pytest.mark.parametrize("equation_form1", ["advective", "continuity"])
+@pytest.mark.parametrize("equation_form2", ["advective", "continuity"])
+def test_coupled_transport_scalar(tmpdir, geometry, equation_form1, equation_form2, tracer_setup):
+    setup = tracer_setup(tmpdir, geometry)
+    domain = setup.domain
+
+    if equation_form1 == "advective":
+        tracer1 = ActiveTracer(name='f1', space='DG',
+                               variable_type=TracerVariableType.density,
+                               transport_eqn=TransportEquationType.advective)
+    else:
+        tracer1 = ActiveTracer(name='f1', space='DG',
+                               variable_type=TracerVariableType.density,
```

Comment:
I suggest that you keep `mixing_ratio` always with `transport_eqn=TransportEquationType.advective` and `density` with `transport_eqn=TransportEquationType.conservative`. Still test a mixture of advective and conservative variables, but since mixing ratios obey the advective form it would be better to keep that in this test